resources/hu/cemu.po

```diff
@@ -57,6 +57,122 @@ msgstr "Írj be egy memória címet"
 msgid "Memory breakpoint"
 msgstr "Memória szünet"
 
+#: CemuApp.cpp
+msgid "Browse"
+msgstr "Tallózás"
+
+#: CemuApp.cpp
+msgid "Select a file"
+msgstr "Válassz ki egy fájlt"
+
+#: CemuApp.cpp
+msgid "Select a directory"
+msgstr "Válassz ki egy könyvtárat"
+
+#: CemuApp.cpp
+msgid "base"
+msgstr "alap"
```

Comment:
Here "base" is a shorthand for "base game".  I would recommend using the extended form for clarity.